pyvisa/ctwrapper/functions.py

```diff
@@ -832,6 +832,19 @@ def get_attribute(library, session, attribute):
         Return value of the library call.
 
     """
+    if attribute not in attributes.AttributesByID:
+        #
+        # the following ResourceAttribute members will fail:
+        #
+        #     fdc_generate_signal_enabled
+        #     interface_parent_number
+        #     pxi_memory_base_bar5
+        #     pxi_memory_size_bar5
+        #     pxi_memory_type_bar5
```

Comment:
Those can be recovered by expanding the range to found on line 3720 of attributes to be `range(0, 6)`

Reply:
Fixed.